macros/dbt_project_evaluator/recursive_dag.sql

```diff
@@ -0,0 +1,6 @@
+{% macro trino__recursive_dag() %}
+{#-- Although Trino supports a recursive WITH-queries,
+-- it is less performant than creating CTEs with loops and unioning them
+-- (549 vs 311 stages in the query if max_depth_dag = 9) --#}
+    {{ return(bigquery__recursive_dag()) }}
```

Comment:
bigquery reference

Reply:
It references [bigquery__recursive_dag](https://github.com/dbt-labs/dbt-project-evaluator/blob/v0.6.2/macros/recursive_dag.sql#L125), which is an implementation which uses loops with CTEs instead of recursive WITH ([default__recursive_dag implementation](https://github.com/dbt-labs/dbt-project-evaluator/blob/v0.6.2/macros/recursive_dag.sql#L5)).
As Trino recursive WITH queries have poor performance, we also want to use loops with CTEs in our implementation.
Although it is named "bigquery", it hasn't any bigquery specific logic and it works correctly with Trino. 
IMHO it is better to just use this macro, instead of copying and pasting entire logic, just to avoid reference to macro named bigquery.

[spark__recursive_dag](https://github.com/dbt-labs/dbt-project-evaluator/blob/v0.6.2/macros/recursive_dag.sql#L237-L240) is implemented in same way - it just reference bigquery__recursive_dag

WDYT?

Reply:
it's fine